src/components/ResumeUploadBtn.js

```diff
@@ -0,0 +1,41 @@
+import React, { useRef, useState } from 'react'
+import { Button } from '../components/Button'
+import { QuestionHeading } from '../components/Typography'
+
+const resumeStyle = {
+  display: 'flex',
+  alignItems: 'center'
+};
+
+const ResumeUploadBtn = () => {
```

Comment:
```suggestion
export default () => {
```

Reply:
thanks for the comments! resolved in latest commit!